### PR TITLE
Destroying a silo give points to marine

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Give 150 supply points when killing a regular silo, 75 when it's a small one

## Why It's Good For The Game

Reward the marines for a successfull silo hunt, i know points are generally not exciting but this is better than nothing. This will give a better sentiment of accomplishment to marines

## Changelog
:cl:
balance: Give 150 supply points when killing a regular silo, 75 when it's a small one
/:cl:

### CODE REVIEW

**Comment by David-Stormwell** [comment]
Don't think this is needed at all, there's already so much of a reason to kill Silos, this just further rewards marines who blitz silos.
Let's say one single marine solo rushed a silo now, packed a flamer and nades.
A easy 100 or 200 points at the cost of one single marine.

**Comment by BraveMole** [comment]
> Don't think this is needed at all, there's already so much of a reason to kill Silos, this just further rewards marines who blitz silos.
> Let's say one single marine solo rushed a silo now, packed a flamer and nades.
> A easy 100 or 200 points at the cost of one single marine.

If marine were able to solo rush silos consistently, there would not be any pool.
I see successfull silo snipe every 10 rounds now, and each time it costs 4-5 marines

**Comment by SpaceLoveSs13** [comment]
200/100 seems a bit too much to me. Atleast 200 should be made 150 imo.

**Comment by BraveMole** [comment]
> 200/100 seems a bit too much to me. Atleast 200 should be made 150 imo.

Eh, we can try that if you want

File: code/__DEFINES/mode.dm [hunk 1]
```diff
@@ -128,6 +128,8 @@
 
 #define SILO_PRICE 1200
 #define XENO_TURRET_PRICE 150
+//Amount of points gave to marine when they destroy a silo
+#define SILO_REWARD 200
```
Comment (BraveMole) [inline code/__DEFINES/mode.dm:132]:
```suggestion
#define SILO_REWARD 150
```

**Comment by TiviPlus** [comment]
Theres enough reason to kill silos imo, this is a little over the top

**Pull request closed without merging**